Demo/Demo/ViewController.swift

```diff
@@ -29,13 +30,30 @@ class ViewController: UIViewController {
     }
 
     func displayDemoFeatureViewController() {
-        let demoViewControllerType = DemoSettings.demoType.viewController
-        let demoViewController = demoViewControllerType.init()
-        demoViewController.modalPresentationStyle = .fullScreen
-
-        navigationItem.title = DemoSettings.demoType.rawValue.uppercased()
-
-        addChild(demoViewController)
-        view.addSubview(demoViewController.view)
+        switch DemoSettings.demoUIFramework {
+        case .uikit:
+            let demoViewController = DemoSettings.demoType.viewController
+            demoViewController.modalPresentationStyle = .fullScreen
+
+            navigationItem.title = DemoSettings.demoType.rawValue.uppercased()
+
+            addChild(demoViewController)
+            view.addSubview(demoViewController.view)
```

Comment:
Not relevant to this PR,
but I just realized everytime we switch demo settings, we'll keep adding childs/subviews to this view controller without ever removing.
Can we remove them before adding new child/subview?
```
children.forEach { viewController in 
  viewController.willMove(toParent: nil)
  viewController.view.removeFromSuperview()
  viewController.removeFromParent()
}
```

we can also make it a function in `extension UIViewController` so we can reuse it :smile: